packages/swingset/.eslintrc.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  root: true,
+  extends: require.resolve('@hashicorp/platform-cli/config/.eslintrc.js'),
+  rules: {
+    '@next/next/no-html-link-for-pages': 'off',
```

Comment:
👀 do you recall why we are turning this off?

Reply:
When I ran `npx eslint .`
I received the following error
`Pages directory cannot be found at /Users/prestonbourne/Desktop/swingset/pages or /Users/prestonbourne/Desktop/swingset/src/pages. If using a custom path, please configure with the `no-html-link-for-pages` rule in your eslint config file.`

Looking at these [docs](https://nextjs.org/docs/messages/no-html-link-for-pages) I assume `es-lint` is expecting a pages directory but we're not using one so I think we want to omit it from the linting